WordPress/src/main/java/org/wordpress/android/util/WPVideoUtils.java

```diff
@@ -0,0 +1,111 @@
+package org.wordpress.android.util;
```

Comment:
This class would benefit from much more commenting. That would make it much easier for another dev to work on it.